library/build.go

```diff
@@ -131,6 +131,7 @@ func (b *Build) Environment(workspace, channel string) map[string]string {
 			// add the tag reference to the list
 			envs["BUILD_TAG"] = tag
 			envs["VELA_BUILD_TAG"] = tag
+			envs["VELA_BUILD_TAG_AUTHOR"] = b.GetSender()
```

Comment:
AFAICT, this `VELA_BUILD_TAG_AUTHOR` environment variable will contain the same value as `VELA_BUILD_SENDER`:

https://github.com/go-vela/types/blob/4dc404f4aabc4ba0f3629113476782f15106085b/library/build.go#L73

Have we considered encouraging usage of `VELA_BUILD_SENDER` instead of `VELA_BUILD_AUTHOR`?

Seems like that would enable the use-case w/o adding another environment variable.

In case my line of questioning seems funny, I'm trying to understand the value of `VELA_BUILD_TAG_AUTHOR` here?

Reply:
Yes that would seem to be the case. An action step moving forward could be to update the docs to note the `tagger` and `sender` connection. I'm fine closing this PR if people agree that should be the way forward -- especially since email doesn't seem to be obtainable.

Reply:
>  I'm fine closing this PR if people agree that should be the way forward 

That would be my vote.

> especially since email doesn't seem to be obtainable.

FWIW, we built a process/API outside of GitHub to map the GitHub username to the underlying Active Directory user to handle this issue.